package.json

```diff
@@ -51,6 +51,7 @@
     "is-relative-url": "^3.0.0",
     "lodash": "^4.17.21",
     "luxon": "^1.24.1",
+    "minimist": "^1.2.6",
```

Comment:
This can probably go under `devDependencies` if it's just for scripts/CLI stuff.

Reply:
Good call, thanks